src/components/structured-text-block/structured-text-block.vue

```diff
@@ -249,13 +262,66 @@
   }
 
   .structured-text__blue-text--center .structured-text__buttons-list {
-    align-self: center;
+    margin-left: auto;
+    margin-right: auto;
   }
 
   .structured-text__image-with-caption {
     margin: var(--spacing-big) 0;
   }
 
+
+  @media (max-width: 719px) {
```

Comment:
Why is this one pixel before the other breakpoint?

Reply:
It's a `max-width` breakpoint. Otherwise they overlap

Reply:
Oh I didn't catch that! Used to only using `min-width` queries, think using a max and min is usually not a good idea, what do you think of refactoring this to `min-width`?

Reply:
Yeah I was in doubt a bit. This is an overwrite of the `margin: var(--spacing-big) 0;` on the default image with caption. > 720 px, I do want the `--spacing-big` though. But I'll go with declaring the `spacing-big` on the default and overwrite it >720 px then again.